azext_vmware/tests/latest/test_vmware_scenario.py

```diff
@@ -83,48 +83,40 @@ class VmwareScenarioTest(AsyncScenarioTest):
-        self.cmd('vmware privatecloud getadmincredentials -g {rg} -n {privatecloud}')
+        self.cmd('vmware privatecloud listadmincredentials -g {rg} -n {privatecloud}')
 
         # add authorization
-        self.cmd('vmware privatecloud addauthorization -g {rg} -n {privatecloud} --authorization-name myauthname')
-        await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
+        # self.cmd('vmware privatecloud addauthorization -g {rg} -n {privatecloud} --authorization-name myauthname')
+        # await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
 
         # delete authorization
-        self.cmd('vmware privatecloud deleteauthorization -g {rg} -n {privatecloud} --authorization-name myauthname')
-        await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
-
-        # add global reach connection
-        self.cmd('vmware privatecloud addglobalreachconnection -g {rg} -n {privatecloud} --id /subscriptions/11234567-0123-0123-0123-0123456789ab/resourceGroups/mygroup/providers/Microsoft.Network/expressroutecircuits/myer/authorizations/myauth --key 00000000-1111-2222-3333-444444444444')
-        await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
-
-        # delete global reach connection
-        self.cmd('vmware privatecloud deleteglobalreachconnection -g {rg} -n {privatecloud} --id /subscriptions/11234567-0123-0123-0123-0123456789ab/resourceGroups/mygroup/providers/Microsoft.Network/expressroutecircuits/myer/authorizations/myauth')
-        await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
+        # self.cmd('vmware privatecloud deleteauthorization -g {rg} -n {privatecloud} --authorization-name myauthname')
+        # await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
 
         # add identity source
-        self.cmd('vmware privatecloud addidentitysource -g {rg} -n {privatecloud} --name groupName --alias groupAlias --domain domain --base-user-dn "ou=baseUser" --base-group-dn "ou=baseGroup" --primary-server ldaps://1.1.1.1:636 --secondary-server ldaps://1.1.1.2:636 --use-ssl true --username someone --credential something')
-        await self.poll_until_result(lambda: self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}'), provissioning_succeeded)
+        # self.cmd('vmware privatecloud addidentitysource -g {rg} -n {privatecloud} --name groupName --alias groupAlias --domain domain --base-user-dn "ou=baseUser" --base-group-dn "ou=baseGroup" --primary-server ldaps://1.1.1.1:636 --secondary-server ldaps://1.1.1.2:636 --use-ssl true --username someone --credential something')

```

Comment:
Adding and deleting an identity source and authorization is to be done using `PrivateClouds_Update` API operation. That work will be in a future pull request.